docs/src/playground/ExamplePage.tsx

```diff
@@ -0,0 +1,11 @@
+import { Grid } from '@mui/material';
+
+export default function ExamplePage() {
+  return (
+    <Grid
+      container
+      columns={{ xs: 2, sm: 4, md: 8, lg: 12 }}
```

Comment:
The columns here are made to match the grid in Figma.  I didn't add the 20px gutter since I wasn't sure if we wanted to apply it on this container or with margin / padding when relevant.